examples/tectonic.openstack.yaml

```diff
@@ -0,0 +1,212 @@
+admin:
+  email: "a@b.c"
+  password: "verysecure"
+  sshKey: "ssh-ed25519 AAAA..."
+openstack:
+  credentials:
+    insecure: false
+
+    cloud:
+
+  baseImage:
+
+  # (optional) Extra AWS tags to be applied to created resources.
+  #
+  # Example: `{ "key" = "value", "foo" = "bar" }`
+  # extraTags:
+
+  # (optional) Name of IAM role to use to access AWS in order to deploy the Tectonic Cluster.
+  # The name is also the full role's ARN.
+  #
+  # Example:
+  #  * Role ARN  = arn:aws:iam::123456789012:role/tectonic-installer
+  # installerRole:
+
+  master:
+    # (optional) This configures master availability zones and their corresponding subnet CIDRs directly.
+    #
+    # Example:
+    # `{ eu-west-1a = "10.0.0.0/20", eu-west-1b = "10.0.16.0/20" }`
+    # customSubnets:
+
+    # Instance size for the master node(s). Example: `m1.medium`.
+    flavorName: m1.medium
+
+    # (optional) List of additional security group IDs for master nodes.
+    #
+    # Example: `["sg-51530134", "sg-b253d7cc"]`
+    # extraSGIDs:
+
+    # (optional) Name of IAM role to use for the instance profiles of master nodes.
+    # The name is also the last part of a role's ARN.
+    #
+    # Example:
+    #  * Role ARN  = arn:aws:iam::123456789012:role/tectonic-installer
+    #  * Role Name = tectonic-installer
+    # iamRoleName:
+
+    rootVolume:
+      # The amount of provisioned IOPS for the root block device of master nodes.
+      # Ignored if the volume type is not io1.
+      iops: 100
+
+      # The size of the volume in gigabytes for the root block device of master nodes.
+      size: 30
+
+      # The type of volume for the root block device of master nodes.
+      type: gp2
+
+  # (optional) If set to true, create private-facing ingress resources (ELB, A-records).
```

Comment:
Do these ELB options work with OpenStack? I don't see where that would happen in this implementation.

Reply:
Good point. LB is not there yet. We'll add the necessary options when we submit the LB PR.